model/src/main/scala/ch.epfl.scala.index.model/release/LanguageVersion.scala

```diff
@@ -34,8 +34,12 @@ final case class ScalaVersion(version: BinaryVersion) extends LanguageVersion {
 }
 
 final case class DottyVersion(version: BinaryVersion) extends LanguageVersion {
-  def family = "dotty"
-  def render: String = s"dotty $version"
+  def family = "3.0"
```

Comment:
I think `scala3` is a better family name than `3.0` because it also contains `0.27` versions.